departures.go

```diff
@@ -38,6 +40,7 @@ func main() {
 	flag.IntVar(&min, "min", 60, "Number of minutes you want to see the departures for")
 	flag.BoolVar(&forceColor, "force-color", false, "Use this flag to enforce color output even if the terminal does not report support")
 	flag.StringVar(&search, "search", "", "Search for the stop name to get the stop ID")
+	flag.StringVar(&stationName, "station", "", "Fetch departures for given station. Ignored if id is provided")
```

Comment:
Could you please capitalize "ID", makes the messages more consistent.

Reply:
My bad. Updated